src/main/java/org/cristalise/kernel/lookup/ItemPath.java

```diff
@@ -67,6 +70,16 @@ public ItemPath(String path) throws InvalidItemPathException {
         checkSysKeyFromPath();
     }
 
+    /**

```

Comment:
This change does not seem to me correct. See my comment on `dependencyMemberPath`

Reply:
This change has been done because of **DependencyDelegate.groovy** line 56.  The issue was if we declare something like this for and Item/Factory in a groovy file:

`        Member(itemPath: "/desc/ActivityDesc/aaa/SiteWorkflow") {
            Property("Version": 0)
        }
`
it throws

`Caught: org.cristalise.kernel.lookup.InvalidItemPathException: Not a valid item path: [desc, ActivityDesc, aaa, SiteWorkflow]
org.cristalise.kernel.lookup.InvalidItemPathException: Not a valid item path: [desc, ActivityDesc, aaa, SiteWorkflow]
	at org.cristalise.kernel.lookup.ItemPath.checkSysKeyFromPath(ItemPath.java:108)
	at org.cristalise.kernel.lookup.ItemPath.<init>(ItemPath.java:70)
	at org.cristalise.dsl.collection.DependencyDelegate.Member(DependencyDelegate.groovy:56)
	at Site$_run_closure14$_closure27.doCall(Site.groovy:283)
`